### PR TITLE
remote_port_description fix

### PR DESCRIPTION
**Pull request by kderynski**
Small fix for mistake in remote_port_description field

### CODE REVIEW

**Comment by mirceaulinic** [comment]
Hey @kderynski could you please provide more information regarding the OS you are using?
On JunOS 13.3 the field `remote_port_description` does not exist (also in the provided mocks can see that: https://github.com/napalm-automation/napalm-junos/blob/develop/test/unit/junos/mock_data/get-lldp-interface-neighbors.txt)

**Comment by mirceaulinic** [comment]
But I see it on JunOS 14.2 though.
You'll probably need to apply the following change:

``` yaml
remote_port_description: {lldp-remote-port-id|lldp-remote-port-description: unicode}
```

**Comment by kderynski** [comment]
Hey @mirceaulinic thanks that you looked at it. I had a feeling that it could not be so easy ;) I have tested it on JunOS 14.1 and it works in this case. I have tried to test change that you have suggested but it doesn't work after brief test. I'll try to test it on different junos versions "old" and "new" tomorrow. I'll back with results :)

**Comment by mirceaulinic** [comment]
Sorry, my suggestion was just theoretical - will check it out tomorrow.

**Comment by mirceaulinic** [comment]
I had a closer look and the field `lldp-remote-port-description` does exist on 13.3 too, but not all the time. I noticed also that even if the port description of the neighbor is set, either the field is not displayed either has the same info as `lldp-remote-port-id`. But there are cases when correctly displays the remote interface description.
Therefore, your correction makes sense.
As previously said, sometimes does not exist for some reasons and if you leave it like now, the value of `remote_port_description` from the output dictionary will be set as `None` instead of unicode (as designed in the model https://github.com/napalm-automation/napalm-base/blob/develop/napalm_base/test/models.py#L87).
I suggest applying one of the following changes on line https://github.com/napalm-automation/napalm-junos/blob/develop/napalm_junos/junos.py#L419

either

``` python
'remote_port_description': (item.remote_port_description if item.remote_port_description is not None else u''),
```

either import [`convert` from napalm_base.helpers](https://github.com/napalm-automation/napalm-base/blob/develop/napalm_base/helpers.py#L170):

``` python
'remote_port_description': convert(unicode, item.remote_port_description),
```

Also I see that https://github.com/napalm-automation/napalm-junos/blob/develop/napalm_junos/junos.py#L416 and https://github.com/napalm-automation/napalm-junos/blob/develop/napalm_junos/junos.py#L418 are the same. Please remove one of them!

Thanks!

File: napalm_junos/utils/junos_views.yml [hunk 1]
```diff
@@ -200,8 +200,7 @@ junos_lldp_neighbors_detail_view:
     parent_interface: {lldp-local-parent-interface-name: unicode}
     remote_port: {lldp-remote-port-id: unicode}
     remote_chassis_id: {lldp-remote-chassis-id: unicode}
-    remote_port: {lldp-remote-port-id: unicode}
```
Comment (mirceaulinic) [inline napalm_junos/utils/junos_views.yml:203]: Maybe I was not able to express myself properly: please do not remove this one - the line that has to be removed is in `junos.py` (will add a note)

File: napalm_junos/junos.py [hunk 2]
```diff
@@ -416,7 +417,7 @@ def get_lldp_neighbors_detail(self, interface=''):
                     'remote_port'               : item.remote_port,
                     'remote_chassis_id'         : item.remote_chassis_id,
                     'remote_port'               : item.remote_port,
```
Comment (mirceaulinic) [inline napalm_junos/junos.py]: This is one of the lines that can be removed

**Comment by kderynski** [comment]
@mirceaulinic ah ok sorry, now I can see it, but I do not quite understand why we need two remote_ports entries in views ?

**Comment by mirceaulinic** [comment]
I do not see two ```remote_ports``` entries in the view?

```yaml
junos_lldp_neighbors_detail_view:
  fields:
    # interface: {lldp-local-port-id: unicode}
    interface_description: {lldp-local-interface: unicode}
    parent_interface: {lldp-local-parent-interface-name: unicode}
    remote_port: {lldp-remote-port-id: unicode}
    remote_chassis_id: {lldp-remote-chassis-id: unicode}
    remote_port_description: {lldp-remote-port-description: unicode}
    remote_system_name: {lldp-remote-system-name: unicode}
    remote_system_description: {lldp-system-description/lldp-remote-system-description: unicode}
    remote_system_capab: {lldp-remote-system-capabilities-supported: unicode}
    remote_system_enable_capab: {lldp-remote-system-capabilities-enabled: unicode}
```

**Comment by mirceaulinic** [comment]
You are right, there are two `remote_port` entries in the view too.

**Comment by mirceaulinic** [comment]
Please remove duplicates from both files. Thanks!

**Comment by mirceaulinic** [comment]
🍾

**Comment by kderynski** [comment]
@mirceaulinic there, should be good now :) Thank you for your help :)

**Pull request merged**